pkg/context/context_test.go

```diff
@@ -63,7 +63,7 @@ func (s *contextSuite) TestCancel(c *check.C) {
 
 func (s *contextSuite) TestCancelCascade(c *check.C) {
 	defer testleak.AfterTest(c)()
-	stdCtx, cancel := context.WithDeadline(context.Background(), time.Now().Add(1*time.Second))
+	stdCtx, cancel := context.WithDeadline(context.Background(), time.Now().Add(1200*time.Millisecond))
```

Comment:
Is it better to set another `startTime2` before this line, and check `time.Since(startTime2)`  in L76 and L78

Reply:
PTAL again